tests/unit/model/designation-test.js

```diff
@@ -2,7 +2,7 @@ import { test, moduleForModel } from 'ember-qunit';
 import Ember from 'ember';
 
 moduleForModel('designation', 'Designation model',{
-  needs: ['model:contact', 'model:organisation', 'model:local_order', 'model:item', 'model:orders_package']
+  needs: ['model:contact', 'model:organisation', 'model:local_order', 'model:item', 'model:orders_package', 'model:location', 'model:code', 'model:donor-condition', 'model:set-item', 'model:packages-location', 'model:image']
```

Comment:
@viveksriram1 , can you please help me to understand one thing?  here you've inject some model that are in snake case latter **model:orders_package** and some are hyphenated **model:donor-condition**.

What is the difference between injecting models using snake case latter and using hyphenated words??
If there is none then can we please follow the convention of using snake case while injecting model??

What do you think?

Reply:
Yeah Sure. I will make the changes.